examples/ssh/dell_idrac_phisical_disks_infos.js

```diff
@@ -0,0 +1,171 @@
+/**
+ * Name: Dell iDRAC Physical Disk Monitoring
+ * Description: Monitors the operational status of physical disks on a Dell server with iDRAC.
+ * 
+ * Communication protocol is SSH.
+ * 
+ * Tested under iDRAC version 7.0.3 21053776 U3 P70
+ * 
+ * Keyboard Interactive option: true/false (depends on iDRAC version).
+ * 
+ * Timeout: should be set to 120 seconds
+ *
+ * Creates a Custom Driver Table with the following columns:
+ *      - FQDD
+ *      - Device Type
+ *      - Device Description
+ *      - Primary Status
+ *      - RAID Status
+ *      - RAID Types
+ *      - Size
+ *      - Used Size
+ *      - Free Size
+ *      - Manufacturer
+ *      - Model
+ *      - Bus Protocol
+ */
+
+// SSH command to retrieve hardware inventory
+var command = "racadm hwinventory";
+
+// SSH options when running the command
+var sshConfig = {
+    "timeout": 100000,
+    "keyboard_interactive": true
+};
+
+// SSH promise definition
+function checkSshError(err) {
+    if(err.message) console.error(err.message);
+    if(err.code == 5) D.failure(D.errorType.AUTHENTICATION_ERROR);
+    if(err.code == 255) D.failure(D.errorType.RESOURCE_UNAVAILABLE);
+    console.error(err);
+    D.failure(D.errorType.GENERIC_ERROR);
+}
+
+// Execute SSH command and return a promise
+function executeCommand(command) {
+    var d = D.q.defer();
+    sshConfig.command = command;
+    D.device.sendSSHCommand(sshConfig, function (output, err) {
+        if (err) {
+            checkSshError(err);
+        } else {           
+            if (output && output.indexOf("COMMAND NOT RECOGNIZED")!==-1) {
+                D.failure(D.errorType.PARSING_ERROR);
+            } else {
+                d.resolve(output);
+            }           
+        }
+    });
+    return d.promise;
+}
+
+// Custom Driver Table to store physical disk information
+var table = D.createTable(
```

Comment:
Please also add value types for the columns:
https://portal.domotz.com/custom-driver/global.html#ValueType

![image](https://github.com/domotz/custom-driver/assets/3306549/d886fdc0-887e-490c-95e5-a4e53d88477c)